dlf/plugins/pageview/tx_dlf_utils.js

```diff
@@ -104,4 +168,55 @@ dlfUtils.setCookie = function(name, value) {
 
     document.cookie = name+"="+escape(value)+"; path=/";
 
+};
+
+/**
+ * Scales down the given features geometrys. as a further improvment this functions
+ * add a unique id to every feature
+ * @param {Array.<ol.Feature>} features
+ * @param {Object} imageObj
+ * @param {number} width
+ * @param {number} height
+ * @return	{Array.<ol.Feature>}
+ */
+dlfUtils.scaleToImageSize = function(features, imageObj, width, height) {
+	
+	// update size / scale settings of imageObj
+	var image;
+    if (width && height) {
+
+    	image = {
+            'width': width,
+            'height': height,
+            'scale': imageObj.width/width,
+        }
+
+    }
+    
+    if (image === undefined)
+    	return [];
+    
+    var scale = image.scale,
+    	height = imageObj.height,
```

Comment:
Here you declare a local variable called "height". There is already a function parameter called "height" which can no longer be accessed (so it is hidden by the new variable). This is confusing for readers of the code, and I expect that static code analyzers will complain.